.changeset/large-dragons-applaud.md

```diff
@@ -0,0 +1,5 @@
+---
+"saleor-dashboard": patch
+---
+
+You can now see the unified look and behavior of pagination action buttons across all list views, including products, customers, orders, and other sections.
```

Comment:
suggestion: can we add a screenshot of this new look? we could add it to our next release notes (can be here on PR description) 🙏🏻